### PR TITLE
Geospatial fixes

### PR DESCRIPTION
**Pull request by jacob6838**
This PR resolves the issues noted in the [WZDx Geospatial Combination PR](https://github.com/CDOT-CV/Work_Zone/pull/49)
- [x] Removing unnecesary monitoring script
- [x] Adding command line arguments to experimental combination scripts
- [x] Adding cmd line arg "updateDates" boolean to experimental combination scripts, to bypass date checks, if set
- [x] Cleaning up experimental_combination attenuator script combination logic
- [x] Cleaning up spelling error caught by cspell

### CODE REVIEW

File: wzdx/experimental_combination/attenuator.py [hunk 1]
```diff
@@ -36,6 +40,30 @@ def main(outputPath="./tests/data/output/wzdx_attenuator_combined.json"):
...
+        "--outputDir", required=False, default="./", help="output directory"
+    )
+    parser.add_argument(
+        "--updateDates",
+        required=False,
+        default="false",
+        help="Boolean (true/false), Update dates to the current date to pass time filter",
+    )
+
+    args = parser.parse_args()
```
Comment (payneBrandon) [inline wzdx/experimental_combination/attenuator.py:63]: With this update do we need to also update our corresponding README documentation? This should be expecting filenames as arguments now but our README still mentions just running `python attenuator.py`

File: wzdx/monitoring/README.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
 # Monitoring Resources
 
-This folder contains scripts that enable monitoring of the production WZDx rest endpoint. These scripts generate GCP custom metrics from the returned WZDx data.
+This folder contains a cloud function which diffs the production and experimental wzdx feeds and outputs them to GCP buckets.
```
Comment (payneBrandon) [inline wzdx/monitoring/README.md]: Is this outputting the diffs to buckets, or are we outputting all feeds to buckets?

**Review by payneBrandon — CHANGES_REQUESTED** [review]
Just a couple questions

File: wzdx/experimental_combination/attenuator.py (hunk 1 above)
Comment (jacob6838) [reply wzdx/experimental_combination/attenuator.py:63]: Updated!

File: wzdx/monitoring/README.md (hunk 2 above)
Comment (jacob6838) [reply wzdx/monitoring/README.md]: Just updated this documentation

Comment (jacob6838) [reply wzdx/monitoring/README.md]: The current production feed, experimental feed, and diff are all stored separately for later analysis

**Review by drewjj — APPROVED** [review]
This looks reasonable to me

File: tests/experimental_combination/integration_testing/main.py [hunk 3]
```diff
@@ -45,23 +44,21 @@
...
 
 def get_wzdx():
-    return [json.loads(open('./tests/data/wzdx.json').read())]
+    return [json.loads(open("./tests/data/wzdx.json").read())]
 
-    query = _datastore_client.query(
-        namespace='rtdh_cache_latest', kind='RTDHCache')
-    query.add_filter('label', '=', 'CacheWZDxPlannedEventsStandard')
-    query.add_filter('condition_1', '=', True)
+    query = _datastore_client.query(namespace="rtdh_cache_latest", kind="RTDHCache")
```
Comment (payneBrandon) [inline tests/experimental_combination/integration_testing/main.py]: Is any of this needed? We return right above this so this code isn't reachable.

File: tests/experimental_combination/integration_testing/settings.py [hunk 4]
```diff
@@ -3,19 +3,18 @@
 
 import os
 
-PROJECT = os.environ.get('GOOGLE_CLOUD_PROJECT', 'cdot-rtdh-dev')
+PROJECT = os.environ.get("GOOGLE_CLOUD_PROJECT", "cdot-rtdh-dev")
```
Comment (payneBrandon) [inline tests/experimental_combination/integration_testing/settings.py]: Should we remove these defaults from the public repo?

File: wzdx/monitoring/experimental/main.py [hunk 5]
```diff
@@ -18,7 +18,7 @@ def get_path():
     date = datetime.datetime.now(pytz.timezone("US/Mountain"))
     print(date)
```
Comment (payneBrandon) [inline wzdx/monitoring/experimental/main.py]: we've got logging imported here, but use quite a few "print" statements. Can we swap that over?

File: tests/experimental_combination/integration_testing/main.py (hunk 3 above)
Comment (jacob6838) [reply tests/experimental_combination/integration_testing/main.py]: removed this whole directory, it is from the rtdh repos, not useful here

File: tests/experimental_combination/integration_testing/settings.py (hunk 4 above)
Comment (jacob6838) [reply tests/experimental_combination/integration_testing/settings.py]: removed this whole directory, it is from the rtdh repos, not useful here

File: wzdx/monitoring/experimental/main.py (hunk 5 above)
Comment (jacob6838) [reply wzdx/monitoring/experimental/main.py]: Updated to logging statements!

File: wzdx/tools/uuid_tools.py [hunk 6]
```diff
@@ -4,7 +4,7 @@
 import uuid
 
 
-CDOT_NAMESPACE_UUID = uuid.UUID('ef39d1dd-53ce-4651-9491-4bba7251a3c0')
+CDOT_NAMESPACE_UUID = uuid.UUID("ef39d1dd-53ce-4651-9491-4bba7251a3c0")
```
Comment (payneBrandon) [inline wzdx/tools/uuid_tools.py]: What is this UUID used for? Is this something we should abstract to the env file?

Comment (jacob6838) [reply wzdx/tools/uuid_tools.py]: This is to seed all of the UUIDs with a shared namespace UUID. This allows all UUIDs generated to be predictable, while also being unique from other users. I moved this to an environment variable.

**Review by payneBrandon — APPROVED** [review]
lgtm!

**Pull request merged**